recipes/bmorph/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: bmorph
+  version: {{ version }}
+
+source:
+  url: https://github.com/UW-Hydro/bmorph/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 57ffdec209f51f4499e20611ee9597c870abb16985f60b929d100302b2203109
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
What's the reason for skipping on Windows? If it's just a missing dependency, we can make this repo `noarch: python` here and just ignore the build failure.

Reply:
Yep, the `mizuroute` dependency doesn't build on Windows. What are the pros/cons of setting `noarch: python` and ignoring the failures vs what I've got set up here?

Reply:
`noarch: python` means only a single package is built that works across all platforms and all versions of Python (since it  only contains Python code). The package will be built on Linux, so the Windows build ends up being immaterial--and it won't even be created when conda-forge turns this recipe into a feedstock.

The alternative is to build a different package for the matrix of every version of Python (currently 3.7, 3.8, 3.9) and architecture/OS (win64, osx64, osx_arm64, linux x86_64, ...). So it's a lot less resources for noarch.

Reply:
Ah, that totally makes sense - I updated the recipe to make this change.

Reply:
@dopplershift - could you let me know if any other changes need to be made on this